nCompiler/R/compile_exprClass.R

```diff
@@ -148,8 +148,8 @@ addIndentToList <- function(x, indent) {
 
 ## error trapping utilities to be used from the various processing steps
 exprClassProcessingErrorMsg <- function(code, msg) {
-    contextCode <- if(!is.null(code$caller)) paste(unlist(nimDeparse(code$caller)), collapse = '\n') else character()
-    ans <- paste0(msg, '\n This occurred for: ', nimDeparse(code),'\n', collapse = '')
+    contextCode <- if(!is.null(code$caller)) deparse(code$caller$Rexpr) else character()
```

Comment:
Same as above: I think we want to use `nDeparse(code)`.  Maybe not.  Let's discuss.

Reply:
Updated, using nDeparse for now: https://github.com/nimble-dev/nCompiler/pull/3/commits/749aaf11b87d1a69a2947ce2c021a997451d061d